content/en/service_owners_guide/overview.md

```diff
@@ -0,0 +1,122 @@
+    {{< nextlink href="https://learn.datadoghq.com/courses/monitoring-k8s-cluster-agent" >}}Monitoring a Kubernetes Cluster{{< /nextlink >}}
+    {{< nextlink href="https://learn.datadoghq.com/courses/dd-api-automation-iac" >}}Datadog API: Automation and Infrastructure as Code{{< /nextlink >}}
+{{< /whatsnext >}} 
+
+**User Interface concentration**:
+{{< whatsnext desc=" " >}}
+    {{< nextlink href="https://learn.datadoghq.com/courses/intro-dashboards" >}}Introduction to Dashboards{{< /nextlink >}}
+    {{< nextlink href="https://learn.datadoghq.com/courses/dashboard-graph-widgets" >}}Discovering Graph Widgets{{< /nextlink >}}
+    {{< nextlink href="https://learn.datadoghq.com/courses/dashboards-slos" >}}Using Dashboards and SLOs{{< /nextlink >}}
+{{< /whatsnext >}}
+
+**Datadog Responders**:
+{{< whatsnext desc=" " >}}
+    {{< nextlink href="https://learn.datadoghq.com/courses/dd-101-sre" >}}Site Reliability Engineer{{< /nextlink >}}
+    {{< nextlink href="https://learn.datadoghq.com/courses/apm-monitors-and-alerting" >}}APM Monitors and Alertin{{< /nextlink >}}
+    {{< nextlink href="https://learn.datadoghq.com/courses/core-web-vitals-lab" >}}Using Datadog RUM to track core web vitals{{< /nextlink >}}
+{{< /whatsnext >}}
+
+**Developer Concentration**:
+{{< whatsnext desc=" " >}}
+    {{< nextlink href="https://learn.datadoghq.com/courses/apm-java-host" >}}Setup APM for Java applications{{< /nextlink >}}
+    {{< nextlink href="https://learn.datadoghq.com/courses/dd-101-dev" >}}Datadog 101: Developer{{< /nextlink >}}
+    {{< nextlink href="https://learn.datadoghq.com/courses/tracking-errors-rum-javascript" >}}Tracking errors with RUM for javascript web applications{{< /nextlink >}}
+{{< /whatsnext >}}
+
+## Create a test environment
+
+After a bit of training, start adding some of your own local conditions. Spend some time installing and experimenting with Datadog in a low-risk sandbox, to help familiarize yourself with the environment. The Datadog installation methods are useful for getting started, and it is always a critical activity for the Datadog service ownership team.  
+
+### Configuring your test environment
+
+#### In-App
+
+The [Datadog UI][8] is also the best place to start when embarking on substantial deployments of the core products. The platform is full of easy to use configuration assistance, live data auto-parsers, contextual auto-suggestions, and many other tools for the admin. The Datadog UI provides the best resources for aiding in some of these tasks. 
+
+A few examples are:
+
+* [APM Service Setup][9]   
+* [Log Pipelines][8]   
+* [Monitor Templates][10] 
+
+#### Host Agent Config Templates
+
+The [Datadog Agent][2] is open-source and published in Github. This is a useful resource for viewing configuration templates and specifications. Also, there are tools for host Agent Fleet Automation. Here are a few examples:
+
+* [Agent Config Template][3]   
```

Comment:
we now have an set of [example config files](https://github.com/DataDog/datadog-agent/tree/main/examples) that could be helpful here

Reply:
added :)